webui/react/src/components/LabelSelectFilter.module.scss

```diff
@@ -3,4 +3,7 @@
     color: rgba(0, 0, 0, 0.65);
     opacity: 1;
   }
+  :global(.ant-select-selection-search-input) {
+    margin-left: 0 !important;
+  }
```

Comment:
**question:** is this property something that should be applicable to all multi-select filter? If so, the better place to put this might be `src/styles/antd.scss`. Also if we do that, is `!important` avoidable?

Reply:
Reviewed. Thanks for the feedback.